.github/workflows/CI.yml

```diff
@@ -17,6 +17,7 @@ jobs:
       fail-fast: false
       matrix:
         version:
+          - '1'
```

Comment:
Awesome, can you please use 1.7 explicitly here so that we officially support 1.6 and 1.7 from now on?

Reply:
Okay, I'll change it that way. (I was just thinking `'1'` would be better because we can notice the test fail before getting new issue like #224.)

Reply:
Yes, I think we want this kind of issue open. It is better to be explicit about the versions we support than assume things are supported in future versions of the language.